R/class-workbook-wrappers.R

```diff
@@ -3118,7 +3118,8 @@ wb_add_thread <- function(
     reply     = FALSE,
     resolve   = FALSE
 ) {
-
+  # wb_add_thread now uses wb_comment internally. No change detected.
+  # means that add_thread does not look at visibility. (I think it's fine.)
```

Comment:
yes, saw that too. visibility is set in wb$comment[[1]]$clientData which is nulled with threaded comments (that's what MS365 does. A comment is added for all spreadsheet software unaware of threaded comments. We can remove this comment in a cleanup commit.

Reply:
Good to know if ever need to revisist